src/pyxtp/pyxtp/__version__.py

```diff
@@ -0,0 +1,3 @@
+__version__ = "0.1.0"
```

Comment:
Should cmake generate that automatically and put the current votca version in here?

Reply:
I am still not sure whether or not we need the `setup.py` and the the content inside `pyxtp`. I need to figure how Python calls the library generated by PyBind11. I think that I am going to delete this file in the future

Reply:
So couldn't we just bind the version of the library here?

Reply:
I have removed all the unnecessary Python code and only left an example